app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -157,9 +179,29 @@ export const ContentForm: React.FC<IContentFormProps> = ({
   );
 
   React.useEffect(() => {
+    const connection = new HubConnectionBuilder()
```

Comment:
Component cannot have api requests within them.  This must be part of a hook as described in my prior comments.

Reply:
Updated, thanks